qfield-plugin-reloader/main.qml

```diff
@@ -74,29 +83,47 @@ Item {
         x: (mainWindow.width - width) / 2
         y: (mainWindow.height - height) / 2
 
+        onAboutToShow: {
+          comboBoxPlugins.currentIndex = comboBoxPlugins.indexOfValue(reloaderSettings.pluginUuid);
+        }
```

Comment:
@mbernasocchi , this is what I was suggesting here -- and since the plugin name is possibly not unique nor static, it's much better to rely on pluginUuid.